docs/source/changelog/0.6.0-changelog.rst

```diff
@@ -0,0 +1,439 @@
+   - Added :func:`~.Mobject.become_points`, which transforms the points, positions and submobjects of a Mobject to match that of the other while keeping style unchanged.
+   - Added example in docs.
+   - Added test.
+
+* `#1363 <https://github.com/ManimCommunity/manim/pull/1363>`__: Change of TeX complier and output file format
+
+
+* `#1359 <https://github.com/ManimCommunity/manim/pull/1359>`__: Make FILE a required argument
+   * Make `FILE` a required argument, `manim/cli/render/commands.py`:L30
+
+* `#1304 <https://github.com/ManimCommunity/manim/pull/1304>`__: Improve Tex string splitting at double braces: only split for double brace groups
+
+
+* `#1340 <https://github.com/ManimCommunity/manim/pull/1340>`__: Add OpenGL support to the new transform animations
+   Made `FadeTransform`, `FadeTransformPieces`, `TransformMatchingShapes` and`TransformMatchingTex` compatible with OpenGL
+
+* `#1343 <https://github.com/ManimCommunity/manim/pull/1343>`__: Make TexTemplate() simple, but keep Tex()'s default template
+   TexTemplate() now returns a simple tex template.
+
+* `#1321 <https://github.com/ManimCommunity/manim/pull/1321>`__: Add OpenGL support to `~.AnimationGroup`
+   Make `~.AnimationGroup` compatible with OpenGL.
+
+* `#1302 <https://github.com/ManimCommunity/manim/pull/1302>`__: Raise appropriate errors in :meth:`~.VMobject.point_from_proportion`
+   - Raise an error if the ``alpha`` argument is not between 0 and 1.
+   - Raise an error if the :class:`~.VMobject` has no points.
+
+* `#1315 <https://github.com/ManimCommunity/manim/pull/1315>`__: Fix performance issues with :meth:`~.VMobject.get_arc_length`, stemming from :pr:`1274`
+
+
+* `#1320 <https://github.com/ManimCommunity/manim/pull/1320>`__: Add `jpeg` extension to the default image extensions
+
+
+* `#1234 <https://github.com/ManimCommunity/manim/pull/1234>`__: New :class:`~.Mobject` method, :meth:`~.Mobject.get_midpoint`
+   Easily get the middle point of a line of an :class:`~.Mobject` with :meth:`~.Mobject.get_midpoint`. (still needs to be adjusted)
+
+* `#1237 <https://github.com/ManimCommunity/manim/pull/1237>`__: Notify user if they are using an outdated version of Manim
+
+
+* `#1308 <https://github.com/ManimCommunity/manim/pull/1308>`__: Improved ManimBanner animations
+
+
+* `#1275 <https://github.com/ManimCommunity/manim/pull/1275>`__: Add SVG <line> element support to :class:`~.SVGMobject`
+
+
+* `#1238 <https://github.com/ManimCommunity/manim/pull/1238>`__: Add parameter `about_point` for `rotate`
+   `about_point` parameter for `rotate`
+
+* `#1260 <https://github.com/ManimCommunity/manim/pull/1260>`__: Change Brace from Tex to SVG (#1258)
+
+
+* `#1122 <https://github.com/ManimCommunity/manim/pull/1122>`__: Support for specifying the interpolation algorithms for individual ImageMobjects
+
+
+* `#1283 <https://github.com/ManimCommunity/manim/pull/1283>`__: Set default value of keyword ``random_seed`` in Scene to None (was 0 and fixed before)
```

Comment:
```suggestion
* `#1283 <https://github.com/ManimCommunity/manim/pull/1283>`__: Set default value of keyword ``random_seed`` in Scene to ``None`` (was 0 and fixed before)
```

Reply:
Fixed the title and also added a reference to :class:`~.Scene`